packages/schematics/angular/application/schema.json

```diff
@@ -113,7 +113,8 @@
     "legacyBrowsers": {
       "type": "boolean",
       "description": "Add support for legacy browsers like Internet Explorer using differential loading.",
-      "default": false
+      "default": false,
+      "x-deprecated": "Legacy browsers support is deprecated since version 12."
```

Comment:
Maybe add a link to the AIO browser support page?
https://angular.io/guide/browser-support